hubble/hubble.go

```diff
@@ -233,6 +241,27 @@ func (k *K8sHubble) Enable(ctx context.Context) error {
 		k.Log("🔑 Found existing CA in secret %s", defaults.CASecretName)
 	}
 
+	if k.params.Wait {
```

Comment:
As it stand we're waiting on the agent to be ready before it is restarted by `k.enableHubble()` that happen on line 265.